kong/db/strategies/cassandra/connector.lua

```diff
@@ -14,7 +14,77 @@ function CassandraConnector.new(kong_config)
   do
     -- Resolve contact points before instantiating cluster, since the
     -- driver does not support hostnames in the contact points list.
+
+    -- TODO: this is an ugly hack to force lua sockets on a third party library
```

Comment:
May be worth being more explicit here:

- we force sync option for resty.dns.client to not spawn a timer
- we force LuaSocket for resty.dns.resolver to not spawn a cosocket in init_by_lua

Can add at merge time, in fact let me take care of it, no worries :+1: